### PR TITLE
Move to using SPDX Licence Format

### PR DESCRIPTION
**Pull request by lukehinds**

### CODE REVIEW

**Comment by lukehinds** [comment]
@jetwhiz , as dicussed

I also changed Nabils email, as it was the old MIT contact address for him. I put the keylime groups io address instead.

**Comment by jetwhiz** [comment]
Should we be on the "BSD-2-Clause" license instead of "MIT"?

**Comment by lukehinds** [comment]
> Should we be on the "BSD-2-Clause" license instead of "MIT"?

Updated

File: keylime/cmd_exec.py [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
-252.227-7013 or 7014 (Feb 2014). Notwithstanding any copyright notice, U.S. Government
-rights in this work are defined by DFARS 252.227-7013 or DFARS 252.227-7014 as detailed
-above. Use of this work other than as specifically authorized by the U.S. Government may
-violate any copyrights that exist in this work.
 '''
-
+SPDX-License-Identifier: BSD-2-Clause
+Copyright 2017 Massachusetts Institute of Technology.
+'''
 import os
```
Comment (jetwhiz) [inline keylime/cmd_exec.py:5]: Do we need a blank line after the header?

File: keylime/ima.py [hunk 2]
```diff
@@ -1,21 +1,6 @@
...
-The software/firmware is provided to you on an As-Is basis
-
-Delivered to the US Government with Unlimited Rights, as defined in DFARS Part
-252.227-7013 or 7014 (Feb 2014). Notwithstanding any copyright notice, U.S. Government
-rights in this work are defined by DFARS 252.227-7013 or DFARS 252.227-7014 as detailed
-above. Use of this work other than as specifically authorized by the U.S. Government may
-violate any copyrights that exist in this work.
+SPDX-License-Identifier: BSD-2-Clause
+Copyright 2017 Massachusetts Institute of Technology.
 '''
```
Comment (jetwhiz) [inline keylime/ima.py:4]: Same blank line after header issue

**Comment by lukehinds** [comment]
@jetwhiz should be good now!

**Pull request merged**